Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/model/ConnectionEndpoint.java

```diff
@@ -72,11 +74,16 @@ public void setServiceUrl(String serviceUrl) {
     }
 
     public String getPingStatus() {
-        return pingStatus;
+        switch (responseCode) {
+            case 0: return "None";
+            case HTTP_OK: return "Pass";
+            default:
+                return "Fail";
+        }
     }
 
-    public void setPing(String pingStatus) {
-        this.pingStatus = pingStatus;
+    public boolean isPass() {
```

Comment:
Can we name this to something a bit more meaningful, such as "isResponding" or "returnedOK"?